go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	golang.org/x/net v0.0.0-20200822124328-c89045814202
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
 	golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd
-	google.golang.org/genproto v0.0.0-20200526211855-cb27e3aa2013
+	google.golang.org/genproto v0.0.0-20200806141610-86f49bd18e98
+	google.golang.org/grpc/examples v0.0.0-20210812181202-a42567fe92f0
```

Comment:
This doesn't seem right to me. `grpc` module should not have a dependency on the `examples` sub-module. If all you are interested in is the the `Echo` server, you could take a look at the `TestService` defined [here](https://github.com/grpc/grpc-go/blob/master/test/grpc_testing/test.proto#L126), which is also used by our existing [end2end_test.go](https://github.com/grpc/grpc-go/blob/master/test/end2end_test.go).

Reply:
Thank you @easwars for the recommendation. I am using TestService now in my end to end tests. It also fixed the test failures.